test/app/modes/C6H6/modes_in.hsd

```diff
@@ -0,0 +1,17 @@
+# Needs the equilibrium geometry, at which the Hessian had been calculated
+Geometry = GenFormat { 
+  <<< geom.gen
+}
+
+RemoveTranslation = Yes
+RemoveRotation = Yes
+
+EigenSolver = DivideAndConquer
```

Comment:
```suggestion
EigenSolver = DivideAndConquer {}
```

Reply:
Not sure about this one, as it's valid (?) HSD and so runs a slightly different code path.

Reply:
Not sure either. If it is valid HSD, let's keep it as is.